jsapp/xlform/src/model.row.coffee

```diff
@@ -28,32 +28,36 @@ module.exports = do ->
     ensureKuid: ->
       if '$kuid' not of @attributes
         @set '$kuid', txtid()
+      return
 
     initialize: ->
       @ensureKuid()
       @convertAttributesToRowDetails()
+      return
 
     isError: -> false
     convertAttributesToRowDetails: ->
       for key, val of @attributes
         unless val instanceof $rowDetail.RowDetail
           @set key, new $rowDetail.RowDetail({key: key, value: val}, {_parent: @}), {silent: true}
+      return
     attributesArray: ()->
       arr = ([k, v] for k, v of @attributes)
       arr.sort (a,b)-> if a[1]._order < b[1]._order then -1 else 1
-      arr
+      return arr
 
     isGroup: ->
-      @constructor.kls is "Group"
+      return @constructor.kls is "Group"
 
     isInGroup: ->
-      @_parent?._parent?.constructor.kls is "Group"
+      return @_parent?._parent?.constructor.kls is "Group"
 
     detach: (opts)->
       if @_parent
         @_parent.remove @, opts
         @_parent = null
       ` `
+      return
```

Comment:
What are those two backticks with a space? Shouldn't we return them? But if they don't do anything, maybe let's remove them.